lib/berlim_web/router.ex

```diff
@@ -17,20 +18,42 @@ defmodule BerlimWeb.Router do
     plug(BerlimWeb.Plugs.RequireAdminAuth)
   end
 
+  pipeline :ensure_user_signed_in do
+    plug(BerlimWeb.Plugs.RequireAuth)
+  end
+
+  pipeline :redirect_logged_user do
+    plug(BerlimWeb.Plugs.RedirectLoggedUser)
+  end
+
   scope "/", BerlimWeb do
+    pipe_through([:browser, :redirect_logged_user])
+
+    get("/", HomeController, :index)
+    get("/sign-in", LoginController, :new)
```

Comment:
da uma olhada no `resources("/sign-in", LoginController, only: [:new, :create])`